src/main/java/ladder/domain/RandomRowGenerator.java

```diff
@@ -0,0 +1,39 @@
+package ladder.domain;
+
+
+import java.security.SecureRandom;
+import java.util.ArrayList;
+import java.util.List;
+
+import static ladder.domain.Line.EXIST;
+import static ladder.domain.Line.NOT_EXIST;
+
+public class RandomRowGenerator implements RowGenerator {
+    private final SecureRandom random = new SecureRandom();
+
+    @Override
+    public List<Line> generate(int size) {
+        List<Line> row = new ArrayList<>();
+
+        for (int index = 0; index < size; index++) {
+            Boolean randomValue = random.nextBoolean();
+            Line cell = Line.getLine(randomValue);
```

Comment:
randomValue보다는 exists 등이 명확할 것 같아요!
wrapper 클래스를 사용할 이유는 없을 것 같네요!
```suggestion
            boolean exists = random.nextBoolean();
            Line cell = Line.getLine(exists);
```